roles/sync/tasks/main.yaml

```diff
@@ -32,7 +32,7 @@
 
 - name: Schedule sync with cron
   ansible.builtin.cron:
-    hour: "2,14"
```

Comment:
Can you revert those values to the originals please since the shorter sync interval is only needed for dhcp.
Maybe you can provide some example values, that you consider good, in the readme.

Reply:
At least in my experience the dns performance was impaired pretty easily, so I would like to keep the sync jobs to a minimum by default.